### ISSUE TITLE
Spot to track entry tickets that are on a char

### ISSUE THREAD

**Issue by TiiaAurora** [TiiaAurora/LS-public-issue-tracker#24]

**Comment by LittleBigDev** [comment]
Also, secret maps.

**Comment by seangay** [comment]
I currently just add these into the custom note for each character..  it works for me so far but somewhere specific would be great

**Comment by LittleBigDev** [comment]
Yes indeed, custom notes are useful for that. Dedicated location would be even better :).

I personally use the notes to remember which character does which Una tasks (to avoid overlaps and reputation progress wastes).
